origin-dapp/src/components/navbar.js

```diff
@@ -37,11 +48,20 @@ class NavBar extends Component {
               ga-label="logo"
             >
               <div className="d-none d-lg-block logo-container">
-                <img
-                  src="images/origin-logo.svg"
-                  className="origin-logo"
-                  alt="Origin Protocol"
-                />
+                {!isWhiteLabel &&
+                  <img
+                    src="images/origin-logo.svg"
+                    className="origin-logo"
+                    alt="Origin Protocol"
+                  />
+                }
+                {isWhiteLabel && logoUrl &&
+                  <img
+                    src={logoUrl}
```

Comment:
There was some discussion about limiting this to `ipfs://` urls (which are then routed to our ipfs gateway) to prevent user tracking. (`file://` URLs would also be ok)

Not a huge concern at first, I think. Worth doing?

Reply:
I think for now we'll want to allow people to use their own external URLs. I think if we did the `ipfs://` only route we'd need to modify the UI to support uploads of images to our IPFS gateway, otherwise it is a lot of friction for users to manually upload their images to IPFS.

IPFS would be nicer and would avoid the problem if invalid/404 URLs and user tracking but it is something we can do post MVP.